cts-java/src/test/java/org/ga4gh/cts/api/endpoints/RawEndpointsIT.java

```diff
@@ -0,0 +1,123 @@
+package org.ga4gh.cts.api.endpoints;
+
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.ga4gh.ctk.transport.TransportUtils;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.net.HttpURLConnection;
+
+import static org.assertj.core.api.StrictAssertions.assertThat;
+
+/**
+ * Tests for the properties of the raw server endpoints.
+ *
+ * @author Herb Jellinek
+ */
+@RunWith(JUnitParamsRunner.class)
+@Category(EndpointsTests.class)
+public class RawEndpointsIT {
+
+    private static final URLMAPPING urls = URLMAPPING.getInstance();
+
+    /**
+     * Given a partial URL, use {@link TransportUtils#makeUrl(String, String)} to combine it with
+     * the root URL and return the result.
+     * @param partialUrl a partial URL
+     * @return a full URL including the root URL
+     */
+    private static String makeFullUrl(String partialUrl) {
+        return TransportUtils.makeUrl(urls.getUrlRoot(), partialUrl);
+    }
+
+    /**
+     * Test that the basic server verbs/methods work as expected.
+     *
+     * @param fullUrl the URL to test
+     */
+    private void testHttpMethods(String fullUrl) throws UnirestException {
```

Comment:
This test is quite opaque to me --- what are we trying to test here?

This might be Java heresy, but it is helpful to break things up into multiple lines sometimes...